openfe/protocols/openmm_md/plain_md_methods.py

```diff
@@ -0,0 +1,604 @@
+        if verbose:
+            logger.info("minimizing systems")
+
+        simulation.minimizeEnergy(
+            maxIterations=simulation_settings.minimization_steps
+        )
+
+        # Get the sub selection of the system to save coords for
+        selection_indices = mdtraj.Topology.from_openmm(
+            simulation.topology).select(simulation_settings.output_indices)
+
+        positions = to_openmm(from_openmm(
+            simulation.context.getState(getPositions=True,
+                                        enforcePeriodicBox=False
+                                        ).getPositions()))
+        # Store subset of atoms, specified in input, as PDB file
+        mdtraj_top = mdtraj.Topology.from_openmm(simulation.topology)
+        traj = mdtraj.Trajectory(
+            positions[selection_indices, :],
+            mdtraj_top.subset(selection_indices),
+        )
+
+        traj.save_pdb(
+            shared_basepath / simulation_settings.minimized_structure
+        )
+        # equilibrate
+        # NVT equilibration
+
+        if verbose:
+            logger.info("Running NVT equilibration")
+
+        # Set barostat frequency to zero for NVT
+        for x in simulation.context.getSystem().getForces():
+            if x.getName() == 'MonteCarloBarostat':
+                x.setFrequency(0)
+
+        simulation.context.setVelocitiesToTemperature(
+            to_openmm(thermo_settings.temperature))
+
+        t0 = time.time()
+        simulation.step(equil_steps_nvt)
+        t1 = time.time()
+        if verbose:
+            logger.info(
+                f"Completed NVT equilibration in {t1 - t0} seconds")
+
+        # Save last frame NVT equilibration
+        positions = to_openmm(
+            from_openmm(simulation.context.getState(
+                getPositions=True, enforcePeriodicBox=False
+            ).getPositions()))
+
+        traj = mdtraj.Trajectory(
+            positions[selection_indices, :],
+            mdtraj_top.subset(selection_indices),
+        )
+        traj.save_pdb(
+            shared_basepath / simulation_settings.equil_NVT_structure
+        )
+
+        # NPT equilibration
+        if verbose:
+            logger.info("Running NPT equilibration")
+        simulation.context.setVelocitiesToTemperature(
+            to_openmm(thermo_settings.temperature))
+
+        # Enable the barostat for NPT
+        for x in simulation.context.getSystem().getForces():
+            if x.getName() == 'MonteCarloBarostat':
+                x.setFrequency(integrator_settings.barostat_frequency.m)
+
+        t0 = time.time()
+        simulation.step(equil_steps_npt)
+        t1 = time.time()
+        if verbose:
+            logger.info(
+                f"Completed NPT equilibration in {t1 - t0} seconds")
+
+        # Save last frame NPT equilibration
+        positions = to_openmm(
+            from_openmm(simulation.context.getState(
+                getPositions=True, enforcePeriodicBox=False
```

Comment:
Hm, I'm not sure, this might depend on what the user wants to do with the trajectory/structure? 
False is the default here and means "the position of each particle will be whatever position is stored in the Context, regardless of periodic boundary conditions". 
So if the users want to e.g. visualize the structure afterwards, they might want to first remove pbc/make things whole. 
Do you have a preference here?

Reply:
We don't really expose this file currently, and it's likely only going to really get used for debugging purposes. I think it's probably fine to leave it as it is.